buildpipeline/pipeline.json

```diff
@@ -163,6 +202,20 @@
         {
           "Name": "Core-Setup-CrossBuild",
           "Parameters": {
+            "PB_DockerTag": "ubuntu1404_cross_prereqs_v2",
+            "PB_Architecture": "arm",
+            "TargetRid": "linux-arm",
+            "PB_PortableBuild": "-portable"
+          },
+          "ReportingParameters": {
+            "OperatingSystem": "Ubuntu 14.04",
```

Comment:
Is it OK that we are building `linux-arm` on `Ubuntu 14.04`?  My assumption was that when we build native code for the `linux` RID, we should be building on RedHat7.2, since that has the lowest supported headers.

/cc @janvorli